skimage/measure/fit.py

```diff
@@ -730,14 +936,27 @@ def ransac(
         ``is_model_valid(model, *random_data)`` and
         ``is_data_valid(*random_data)`` must all take each data array as
         separate arguments.
-    model_class : object
-        Object with the following object methods:
+    model_class : type
+        Class with the following methods:
 
-         * ``success = estimate(*data)``
-         * ``residuals(*data)``
+        * Either:
+
+          * ``from_estimate`` class method returning transform instance, as in
+            ``tform = model_class.from_estimate(*data)``; the resulting
+            ``tform`` should be truthy (``bool(tform) == True``) where
```

Comment:
Should we add the check we recommend here?

```
if tform: ...
```

Reply:
Hmm - how to convey that in the docstring?   Put an embedded code example in there?   Can we assume that people prepared to make custom classes won't be much distracted by the `bool(tform)` notation here?